assets/js/main.js

```diff
@@ -1,15 +1,14 @@
 async function getMovies() {
-  const result = await fetch("http://localhost:8080/");
+  const result = await fetch("https://api.themoviedb.org/3/movie/top_rated?api_key=9f120f0e95e5cedf05a78abda9ef8477&language=en-US&page=1");
   const resultJson = await result.json();
   const {results: movies} = resultJson;
   const movieList = document.getElementById("movie-list");
-  const imgSrc = "./assets/test.jpg";
   let listHtml = "";
 
   movies.forEach(movie => {
-    listHtml += `<li class="movie-item">
-    <a href="./movie.html">
-      <img id="img" class="img" src=${imgSrc}>
+    listHtml += `<li id="movie-item" class="movie-item">
```

Comment:
Remove all `id`s that are defined in the loop. The reason is that then you will have multiple identical ids in the DOM. Which is not valid.

Reply:
Done.